torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -72,7 +72,7 @@
 )
 from .wrap import (
     HandleInitMode,
-    ParamExecOrderPolicy,
+    ParamExecOrderWrapPolicy,
```

Comment:
Do you feel strongly about including `Wrap` in the name? I guess it makes sense since the FSDP argument is `auto_wrap_policy`, but it is becoming a mouthful to say param - exec - order - wrap policy 😆 (and unlike the others, it does not recursive wrap).